packages/docs/src/components/ExampleBox.tsx

```diff
@@ -44,6 +47,14 @@ export function ExampleBox({
     </>
   );
 
+  const NotDetectedOverlayDefault = (
+    <Stack css={styles.overlay} justify="center" align="center">
+      <Heading as="h6">Wallet not detected</Heading>
+      <Text>Please install the Fuel Wallet to use this demo.</Text>
+      {downloadContent}
+    </Stack>
```

Comment:
As the space is small and the message can be repetitive maybe just add:

`Wallet not detected`
`Add a link to install page`